backend/core/settings.py

```diff
@@ -235,3 +235,7 @@
         "debug_toolbar.panels.redirects.RedirectsPanel",
         "debug_toolbar.panels.profiling.ProfilingPanel",
     ]
+
+
+# Configuration to enable elastic search logic in views.py
+ENABLE_ELASTIC = "OFF"
```

Comment:
Can we make this an environment variable rather than having it be hard coded?
Also can this be a boolean instead of a string?